crates/rpc-replay/src/block_context.rs

```diff
@@ -49,16 +56,21 @@ pub fn build_block_context(
         use_kzg_da,
     };
 
+    let strk_fee_token_address = get_env_var_or_default(
+        "SNOS_STRK_FEE_TOKEN_ADDRESS",
+        "0x04718f5a0fc34cc1af16a1cdee98ffb20c31f5cd61d6ab07201858f4287c938d",
+    );
+    let eth_fee_token_address = get_env_var_or_default(
+        "SNOS_ETH_FEE_TOKEN_ADDRESS",
+        "0x049d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7",
```

Comment:
Consider creating constants for these, something like:
```
const DEFAULT_ETH_FEE_TOKEN_ADDRESS = ....
```

Reply:
Valid, added.